MobileFilter/sections/css_extended.txt

```diff
@@ -6,7 +6,7 @@
 ! Bad: example.org##main > div.ad (should be in specific.txt)
 !
 !
-xhamster.com#?#div[class]:has(> div[class*="widget"])
+!xhamster.com#?#div[class]:has(> div[class*="widget"])
```

Comment:
We do comment on the rules, but it's really rare. It's better to remove the rule, but before that could you show in the screenshot or video how the rule have been working incorrectly?
I can't reproduce it on my end.